ilastik/applets/dataSelection/dataSelectionGui.py

```diff
@@ -430,6 +431,11 @@ def getImageFileNamesToOpen(cls, parent_window, defaultDirectory):
 
             if file_dialog.exec_():
                 fileNames = file_dialog.selectedFiles()
+                # For the n5 extension the attributes.json file has to be selected in the file dialog.
+                # However we need just the *.n5 directory-file.
+                for i in range(len(fileNames)):
+                    if os.path.join(".n5", "attributes.json") in fileNames[i]:
```

Comment:
This will not fire if one selects an "attributes.json" of a dataset, e.g. in `blah.n5/exported_data/attributes.json`